src/Microsoft.PowerShell.Commands.Utility/commands/utility/NewGuidCommand.cs

```diff
@@ -0,0 +1,88 @@
+
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+	/// <summary>
+	/// The implementation of the "new-alias" cmdlet
+	/// </summary>
+	/// 
+	[Cmdlet(VerbsCommon.New, "Guid", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=526920", DefaultParameterSetName = "Guid")]
+	[OutputType(typeof(Guid))]
+	public class NewGuidCommand : Cmdlet
+	{
+		/// <summary>
+		/// True to return the Guid.Empty
+		/// </summary>
+		[Parameter(ParameterSetName = "Empty")]
+		public SwitchParameter Empty { get; set; }
+
+		/// <summary>
+		/// The number of guids to generate
+		/// </summary>
+		[Parameter(ParameterSetName = "Count")]
+		[ValidateRange(1, int.MaxValue)]
+		public int Count { get; set; }
+
+		/// <summary>
+		/// A guid in string format to be converted to guids
+		/// </summary>
+		[Parameter(ValueFromPipeline=true, ValueFromPipelineByPropertyName = true, ParameterSetName = "Guid", Position = 1)]
+		public string[] Guid{ get; set; }
+
+		/// <summary>
+		/// returns a guid for each pipeline item
+		/// </summary>
+		protected override void ProcessRecord()
+		{
+			if (Empty)
+			{
+				WriteObject(System.Guid.Empty);
+				return;
+			}
+
+			if (Count > 0)
+			{
+				for (int i = 0; i < Count; ++i)
+				{
+					WriteObject(System.Guid.NewGuid());
+				}
+				return;
+			}
+
+			if (Guid == null)
+			{
+				WriteObject(System.Guid.NewGuid());
+				return;
+			}
+			foreach (var stringGuid in Guid)
```

Comment:
This code feels like it doesn't belong in a cmdlet with the verb `New`, and I don't think we want a cmdlet `Convert-ToGuid`.